### PR TITLE
Switch to mono repo

### PR DESCRIPTION
**Pull request by cabhishek**
- Restructure app to be a mono repo using [learna](https://github.com/lerna/lerna)
- Use [prettier](https://github.com/jlongster/prettier) for code style consistency
- Update travis.yml and readme details

### CODE REVIEW

**Comment by rgbkrk** [comment]
Oh nice you're trying prettier here too!

**Comment by jdetle** [comment]
👍  Thanks @cabhishek. Will be cool to learn lerna through this repo.

File: packages/commuter-server/config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { bucket = null, pathDelimiter = "/", basePath = null } = process.env;
```
Comment (rgbkrk) [inline packages/commuter-server/config.js]: Small nit: I'd probably make these upper case with `_` like `PATH_DELIMITER` as well as prefixed. `COMMUTER_BUCKET`, `COMMUTER_PATH_DELIMITER`, and `COMMUTER_BASEPATH`. It makes sysops folks much happier.

Comment (cabhishek) [inline packages/commuter-server/config.js]: Ok. Will make it upper case

**Pull request merged**